Shared/database.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 
-include_once(__DIR__ . "/../../coursesyspw.php");
```

Comment:
This line is important that it is not changed, this filepath is not relative on purpose to make it so that it is always referencing the correct file even when included into services.